static/js/components/dashboard/FinalExamCard.js

```diff
@@ -191,7 +227,15 @@ export default class FinalExamCard extends React.Component<void, Props, void> {
     case PEARSON_PROFILE_INVALID:
       return invalidCard(navigateToProfile);
     case PEARSON_PROFILE_SCHEDULABLE:
-      return schedulableCard(profile, program, navigateToProfile, submitPearsonSSO, pearson);
+      return schedulableCard(
+        profile,
+        program,
+        navigateToProfile,
+        pearson,
+        showPearsonTOSDialog,
+        pearsonTosDialogVisible,
```

Comment:
can we consistently capitalize `TOS`? I'm not sure what's best, I would suggest standardizing on either `TOS` or `ToS` though (here we have both `Tos` and `TOS` right now).